storage/innobase/fil/fil0fil.cc

```diff
@@ -10870,7 +10871,7 @@ byte *fil_tablespace_redo_extend(byte *ptr, const byte *end,
   }
 
   /* Offset within the file to start writing zeros */
-  os_offset_t offset = mach_read_from_8(ptr);
+  // os_offset_t offset = mach_read_from_8(ptr);
```

Comment:
Why is this commented out? 
Not used ?

Reply:
yes. it not used. throwing compilation error.